src/controllers/words.js

```diff
@@ -46,6 +46,12 @@ export const searchWordUsingEnglish = async ({ query, searchWord, ...rest }) =>
 /* Gets words from MongoDB */
 export const getWords = async (req, res, next) => {
   try {
+    const hasQuotes = req.query.keyword
```

Comment:
nit: this can simplified to:
```js
const hasQuotes = req.query.keyword && req.query.keyword.match(/["'].*["']/) !== null);
```
if `req.query.keyword` doesn't exist, this conditional will short circuit and return a falsely value.